redshift_sqlalchemy/dialect.py

```diff
@@ -28,39 +29,87 @@ class RedShiftDDLCompiler(PGDDLCompiler):
     Table level properties can be set using the dialect specific syntax. For
     example, to specify a distkey and style you apply the following ::
 
-        table = Table(metadata,
-                      Column('id', Integer, primary_key=True),
-                      Column('name', String),
-                      redshift_diststyle="KEY",
-                      redshift_distkey="id"
-                      redshift_sortkey=["id", "name"]
-                      )
+    >>> import sqlalchemy as sa
+    >>> from sqlalchemy.schema import CreateTable
+    >>> engine = sa.create_engine('redshift+psycopg2://example')
+    >>> metadata = sa.MetaData()
+    >>> user = sa.Table(
+    ...     'user',
+    ...     metadata,
+    ...     sa.Column('id', sa.Integer, primary_key=True),
+    ...     sa.Column('name', sa.String),
+    ...     redshift_diststyle='KEY',
+    ...     redshift_distkey='id',
+    ...     redshift_sortkey=['id', 'name'],
+    ... )
+    >>> print(CreateTable(user).compile(engine))
+    <BLANKLINE>
+    CREATE TABLE "user" (
+        id INTEGER NOT NULL,
+        name VARCHAR,
+        PRIMARY KEY (id)
+    ) DISTSTYLE KEY DISTKEY (id) SORTKEY (id, name)
```

Comment:
I'm of the opinion that these table attributes should be separated by newlines, but that would also be a change for a separate PR.